src/apis/index.js

```diff
@@ -2,6 +2,7 @@
 import ajax from '../helpers/ajax';
```

Comment:
Since, `ajax` is not being used. Can we remove it from the imports?

Reply:
> Since, `ajax` is not being used. Can we remove it from the imports?
No, @arteevraina we can't remove import ajax statement since other functions in the code are using it. I think the pr genrated is absolutly fine.

Reply:
@927tanmay Yes, I was wrong.